schemas/panels/time-series/time-series.cue

```diff
@@ -17,9 +17,12 @@ import (
 	"github.com/perses/perses/schemas/common"
 )
 
+#legend_value: "AverageNonNull" | "FirstNonNull" | "LastNonNull" | "Min" | "Max" | "Total"
```

Comment:
I prefer `calculation` to be consistent with GaugeChart and StatChart

Reply:
Could be good to add these options as supported calculations in those other panel types too in the future so same types / utils can be reused